schema/kea.yml

```diff
@@ -0,0 +1,130 @@
+---
+name: kea
+reconfigureEndpoint: "/kea/service/reconfigure"
+resources: 
+  - name: KeaConfig
+    filename: kea_config
```

Comment:
For these resources, can you remove the `Kea` prepended to the name. It's just a style preference, but since each resource is in it's own package, and accessed through the controller like `Kea().Get<resource>()`, there's no need to have that extra reference.